tools/gromacs/npt.xml

```diff
@@ -0,0 +1,180 @@
+<tool id="gmx_npt" name="Gromacs NPT" version="@VERSION@">
```

Comment:
please add for every tool a short description and make sure the `name` is also meaningful. The purpose of `name`+`description` is that in the Galaxy toolbox this is shown. Have a look at usegalaxy.eu how this is done usually. You have a short name, with a slightly longer description.

NPT is just not very intuitive. You probably also want to mention the name gromacs maybe?